src/StudentSystemServices/Shibboleth/Controllers/ShibbolethController.php

```diff
@@ -137,6 +137,10 @@ private function getServerVariable($variableName)
 
         $variable = Request::server($variableName);
 
+        if($variableName === 'iTrustSuppress') {
```

Comment:
@bwharrisUIC It looks like `$variableName` ultimately comes from the values a user will set in the `config/shibboleth.php` file. Out of precaution, can you make this check case insensitive? Just in case.